doc/user-guide/Push-notifications.md

```diff
@@ -133,19 +135,19 @@ See the diagram to examine the process described in this section along with the
 
 ### Registering with a Push Service provider
 
-Firstly, the client application has to get a device-specific token from Push Service Provider (FCM or APNS). 
+Firstly, the client application has to get a device-specific token from Push Service Provider (FCM or APNS).
 This process is different, depending on the platform, so please consult your Push Service Provider's manual to see how to get this token.
 For example, [here](https://firebase.google.com/docs/cloud-messaging/android/client) you can learn about setting up _FCM_ on _Android_ platform and [here](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#/apple_ref/doc/uid/TP40008194-CH6-SW1) you can learn about setting up _APNS_ on _iOS_ platform.
 
 After this step, your application shall be able to receive _FCM_ or _APNS_ token - it will be required in the next step of this tutorial.
 
 ### Setting up an XMPP `pubsub` node
 
-This first step unfortunately is specific to the `app-server` your client application uses. 
-In case of MongooseIM, you just need to allocate (create) special `PubSub` node that will act as gateway for all notifications sent by the XMPP chat server. 
+This first step unfortunately is specific to the `app-server` your client application uses.
+In case of MongooseIM, you just need to allocate (create) special `PubSub` node that will act as gateway for all notifications sent by the XMPP chat server.
```

Comment:
+In case of MongooseIM, you just need to allocate (create) a special `PubSub` node that will act as a gateway for all notifications sent by the XMPP chat server.